### PR TITLE
Update documentation about tokens

### PR DESCRIPTION
**Pull request by Pessimistress**

### CODE REVIEW

**Review by chrisgervang — APPROVED** [review]

File: docs/get-started/mapbox-tokens.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-Depending on which Mapbox GL JS version (or fork) you use, you may need a Mapbox token. You will need a Mapbox token if you use:
+`react-map-gl` itself is open source and free. It provides a React wrapper for `mapbox-gl`, `maplibre-gl` or compatible projects.
+Depending on which base map library you use, you may need a Mapbox token. You will need a Mapbox token if you use:
 
 -  [mapbox-gl@>=2.0.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v2.0.0) and above - requires a mapbox access token in order to access the map renderer, and generates billable events regardlesss of whether you are displaying your own maps.
--  `mapbox-gl@1.x` or [maplibre-gl](https://github.com/maplibre/maplibre-gl-js) - requires an access token only if you load the map styles and tiles from Mapbox's data service. See "Display Maps Without A Mapbox Token" section below for using non-Mapbox tiles.
+-  `mapbox-gl@1.x` or `maplibre-gl` - requires an access token only if you load the map styles and tiles from Mapbox's data service. See "Display Maps Without A Mapbox Token" section below for using non-Mapbox tiles.
```
Comment (HarelM) [inline docs/get-started/mapbox-tokens.md:7]: MapLibre 2.x removed the mapbox token. I seem to recall a plugin was made to support mapbox token in new versions.

File: docs/get-started/mapbox-tokens.md [hunk 2]
```diff
@@ -18,22 +18,24 @@ We recommend using an environment variable to minimize leaking risks. See [secur
...
 
+- Use [maplibre-gl](https://maplibre.org/maplibre-gl-js-docs/api/). See [Get Started](./get-started.md) for how to configure your project.
 - Stay on `mapbox-gl@1.x`. react-map-gl plans to continue supporting this release in the foreseeable future, however, this version will not include any of the latest features of the map renderer, nor get any future updates from Mapbox.
-- Use a community fork of mapbox-gl, for example [maplibre-gl](https://www.npmjs.com/package/maplibre-gl). See [Get Started](./get-started.md) for how to configure your project.
 
-To use your own map service, you will need a custom Mapbox GL style that points to your own [vector tile source](https://www.mapbox.com/mapbox-gl-js/style-spec/), and pass it to `Map` using the `mapStyle` prop. This custom style must match the schema of your tile source.
+To use your own map service, you will need a custom map style that points to your own [tile source](https://maplibre.org/maplibre-style-spec/), and pass it to `Map` using the `mapStyle` prop.
```
Comment (HarelM) [inline docs/get-started/mapbox-tokens.md:26]: Shouldn't the link be to the source page in the style spec docs?

File: docs/get-started/mapbox-tokens.md [hunk 3]
```diff
@@ -18,22 +18,24 @@ We recommend using an environment variable to minimize leaking risks. See [secur
...
-Open source tile schemas include:
+Some alternative map data providers include:
 
-- [TileZen schema](https://tilezen.readthedocs.io/en/latest/layers/)
-- [OpenMapTiles schema ](https://openmaptiles.org/schema/)
+- [MapTiler](https://www.maptiler.com)
+- [Amazon Location Service](https://docs.aws.amazon.com/location/latest/developerguide/map-concepts.html)
 
-Some useful resources for creating your own map service:
+Useful resources for creating your own map tiles and service:
```
Comment (HarelM) [inline docs/get-started/mapbox-tokens.md:33]: Pmtiles is a great and easy way to host tiles using a single zip file.
It needs the relevant plugin for maplibre though.

File: docs/get-started/mapbox-tokens.md [hunk 4]
```diff
@@ -18,22 +18,24 @@ We recommend using an environment variable to minimize leaking risks. See [secur
...
 
-- [Mapbox Vector Tile Spec](https://www.mapbox.com/developers/vector-tiles/)
-- [Open source tools](https://github.com/mapbox/awesome-vector-tiles)
+- [Martin Tile Server](https://maplibre.org/martin/introduction.html)
+- [OpenMapTiles](https://openmaptiles.org)
+- [Mapbox Vector Tile Spec](https://github.com/mapbox/vector-tile-spec)
+- [List of open source tools](https://github.com/mapbox/awesome-vector-tiles)
```
Comment (HarelM) [inline docs/get-started/mapbox-tokens.md:38]: Maybe add a link here to awesome-maplibre repo?

Comment (Pessimistress) [reply docs/get-started/mapbox-tokens.md:38]: That repo seems to be generic maplibre add-ons instead of tile creation/server tools?

Comment (HarelM) [reply docs/get-started/mapbox-tokens.md:38]: Yes, it's not specific.

**Pull request merged**